src/InboundServer/handlers.js

```diff
@@ -146,13 +168,19 @@ const postQuotes = async (ctx) => {
     // so it is safe to return 202
     ctx.response.status = 202;
     ctx.response.body = '';
+    histTimerEnd({ success: true });
 };
 
 
 /**
  * Handles a POST /transfers request
  */
 const postTransfers = async (ctx) => {
+    const histTimerEnd = Metrics.getHistogram(
+        'inbound_post_transfers',
```

Comment:
Any reason that you are not using the information that you set in the Route Map instead....it seems that this is using the same metric names for the event framework?

Reply:
decided not to go with this option as name in route map requires the path to access